### PR TITLE
Rework DAL baker tutorial

### PR DESCRIPTION
**Pull request by timothymcmackin**
We're going to keep this DAL baker tutorial along with the topic on [Running a DAL attester node](https://tezos.gitlab.io/shell/dal_run.html) even though they overlap because we want to get people running DAL nodes and some might like a complete guide for doing it from scratch.

Takes over on #455.

Preview: 
https://docs-staging-git-rework-dal-baker-tutorial-trili-tech.vercel.app/tutorials/join-dal-baker

- Refer to this as a guide because it's not really a tutorial
- Focus on Ghostnet but provide info for other networks
- Link to the Octez docs for people who are more familiar with the process
- Move it to the section for "Intermediate" tutorials, rather than "Advanced"
- Add installation details
- Clarify introduction to send current bakers to Octez docs and to explain that this tutorial starts from scratch

### CODE REVIEW

File: docs/tutorials/join-dal-baker/run-node.md [hunk 1]
```diff
@@ -2,58 +2,79 @@
...
+      tar xf octez-binaries-20.3-linux-x86_64.tar.gz
+      sudo cp octez-x86_64/octez* /usr/local/bin/
+      ```
+
+- For Weeklynet, look up the necessary version of Octez at https://teztnets.com/weeklynet-about and install it with the instructions there.
+
+For more installation options, see [Installing Octez](https://tezos.gitlab.io/introduction/howtoget.html) in the Octez documentation.
 
-1. Install the Octez suite of programs, including `octez-client`, `octez-node`, and `octez-dal-node` as described in [Installing Octez](https://tezos.gitlab.io/introduction/howtoget.html) in the Octez documentation.
 If you build from source, you can use the `latest_branch` branch to work with Ghostnet.
```
[the remark this answers is not in the document]
Comment (timothymcmackin) [reply docs/tutorials/join-dal-baker/run-node.md:35]: Funny, I don't see a `latest_release` branch or tag, either. I'll remove this item.

[the remark this answers is not in the document]
Comment (NicNomadic) [reply docs/tutorials/join-dal-baker/run-node.md]: It uses a hyphen, not an underscore: `latest-release`

[the remark this answers is not in the document]
Comment (timothymcmackin) [reply docs/tutorials/join-dal-baker/run-node.md]: Ah, thanks. Changed it to the `latest-release` branch.

File: docs/tutorials/join-dal-baker.md [hunk 2]
```diff
@@ -2,47 +2,57 @@
...
 authors: Tezos core developers, Tim McMackin
 last_update:
-  date: 19 August 2024
+  date: 21 October 2024
 ---
 
-The Tezos data availability layer (DAL) is a key component for the scalability of Tezos.
-In a nutshell, the DAL increases the data bandwidth available for Tezos Smart Rollups by providing a peer-to-peer network that they can use to fetch data without compromising security.
+The Tezos data availability layer (DAL) is a peer-to-peer network that Tezos Smart Rollups can use to fetch data securely.
+The DAL is a key component for the scalability and bandwidth of Tezos and it's important for bakers to run DAL nodes along with their layer 1 nodes to support it.
```
[the remark this answers is not in the document]
Comment (timothymcmackin) [reply docs/tutorials/join-dal-baker.md]: Removed ambiguous "to support it."

**Pull request merged**